features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml

```diff
@@ -31,4 +31,7 @@
     <Scope id="address" displayName="Address" description="View the address of the user.">
         <Claim>address,street_address,locality,region,postal_code,country</Claim>
     </Scope>
+    <Scope id="groups" displayName="Groups" description="View information related to groups of the user.">
```

Comment:
Should we say View the groups the user belongs to or something similar here?

Reply:
what about `View what groups the user belongs to` ?

Reply:
View groups the user belongs to